service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.fpl.handlers;
+
+import com.google.common.collect.ImmutableMap;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.fpl.events.SubmittedCaseEvent;
+import uk.gov.hmcts.reform.fpl.service.EmailLookUpService;
+import uk.gov.hmcts.reform.fpl.service.NotificationService;
+import uk.gov.service.notify.NotificationClientException;
+
+import java.util.Map;
+
+@Component
+public class NotificationHandler {
+
+    private final EmailLookUpService emailLookUpService;
+    private final NotificationService notificationService;
+
+    @Autowired
+    public NotificationHandler(EmailLookUpService emailLookUpService, NotificationService notificationService) {
+        this.emailLookUpService = emailLookUpService;
+        this.notificationService = notificationService;
+    }
+
+    @EventListener
+    public void sendNotificationToHmctsAdmin(SubmittedCaseEvent event) {
+        CaseDetails caseDetails = event.getCallbackRequest().getCaseDetails();
+        String localAuthorityCode = caseDetails.getData().get("caseLocalAuthority").toString();
```

Comment:
I prefer to cast to String instead of calling `.toString()`. Value should be always string.